<?xml version="1.0"?>
<launch>
  <param name="use_sim_time" value="false"/>

  <node name="sophon_robot" pkg="sophon_robot" type="sophon_robot_node" output="screen" respawn="true">
    <param name="port_name" value="/dev/ttyUSB0"/>
    <param name="linear_correction" value="1.0"/>
    <param name="angular_correction" value="1.0"/>
    <param name="publish_odom_transform" value="true"/>
  </node>

  <node pkg="tf" type="static_transform_publisher" name="base_link_to_imu" args="0 0 0.07 0 0 0 base_link base_imu_link 20"/>
</launch>
